### PR TITLE
feat: set fastq screen & kraken db shared resource paths depending on HPC

### LINKED ISSUE #64: biowulf-specific mount paths do not work on FRCE
When running the test dataset on FRCE, jobs fail because of biowulf-specific mount paths:

```
FATAL:   container creation failed: mount /data/CCBR_Pipeliner->/data/CCBR_Pipeliner error: while mounting /data/CCBR_Pipeliner: mount source /data/CCBR_Pipeliner doesn't exist
```

This occurs with renee v2.3 and 2.5.8

### PR DESCRIPTION
**Pull request by kelly-sovacool**
## Changes

Detect the HPC and use a different fastq screen config file and kraken db path depending on if it's Biowulf or FRCE. Still allow users to override these with `--shared-resources` flag

#### Biowulf

```sh
RENEE_DIR=/data/CCBR_Pipeliner/Pipelines/RENEE/renee-dev-sovacool
WORKDIR=/data/sovacoolkl/renee_test_iss-64
rm -rf $WORKDIR

module load ccbrpipeliner
$RENEE_DIR/bin/renee run \
    --input $RENEE_DIR/.tests/*.R?.fastq.gz \
    --output $WORKDIR \
    --genome hg38_30 \
    --mode slurm \
    --sif-cache /data/CCBR_Pipeliner/SIFS
```

#### FRCE

```sh
#!/bin/bash
RENEE_DIR=/mnt/projects/CCBR-Pipelines/pipelines/RENEE/renee-dev-sovacool
WORKDIR=/scratch/cluster_scratch/$USER/renee_test_iss-64
rm -rf $WORKDIR

module load singularity
$RENEE_DIR/bin/renee run \
	--input $RENEE_DIR/.tests/*.R?.fastq.gz \
	--output $WORKDIR \
	--tmp-dir $WORKDIR \
	--genome hg38_30 \
	--mode slurm \
	--sif-cache /mnt/projects/CCBR-Pipelines/SIFs
```

## Issues

Fixes #64 

## PR Checklist

(~Strikethrough~ any points that are not applicable.)

- [x] This comment contains a description of changes with justifications, with any relevant issues linked.
- ~[ ] Update docs if there are any API changes.~
- [x] Update `CHANGELOG.md` with a short description of any user-facing changes and reference the PR number. Guidelines: https://keepachangelog.com/en/1.1.0/

### CODE REVIEW

**Comment by kelly-sovacool** [comment]
waiting on #85 to be merged so I can test this again with those fixes incorporated

Test runs worked.

**Comment by kelly-sovacool** [comment]
@kopardev just waiting on you to review this before we can merge it.

**Review by slsevilla — APPROVED** [review]
looks good!

**Pull request merged**
